src/o365/graph/commands/groupsetting/groupsetting-set.spec.ts

```diff
@@ -0,0 +1,434 @@
+    sinon.stub(request, 'get').callsFake((opts) => {
+      if (opts.url === `https://graph.microsoft.com/v1.0/groupSettings/c391b57d-5783-4c53-9236-cefb5c6ef323`) {
+        return Promise.resolve({
+          "id": "c391b57d-5783-4c53-9236-cefb5c6ef323", "displayName": null, "templateId": "62375ab9-6b52-47ed-826b-58e47e0e304b", "values": [{ "name": "CustomBlockedWordsList", "value": "" }, { "name": "EnableMSStandardBlockedWords", "value": "false" }, { "name": "ClassificationDescriptions", "value": "" }, { "name": "DefaultClassification", "value": "" }, { "name": "PrefixSuffixNamingRequirement", "value": "" }, { "name": "AllowGuestsToBeGroupOwner", "value": "false" }, { "name": "AllowGuestsToAccessGroups", "value": "true" }, { "name": "GuestUsageGuidelinesUrl", "value": "" }, { "name": "GroupCreationAllowedGroupId", "value": "" }, { "name": "AllowToAddGuests", "value": "true" }, { "name": "UsageGuidelinesUrl", "value": "" }, { "name": "ClassificationList", "value": "" }, { "name": "EnableGroupCreation", "value": "true" }]
+        });
+      }
+
+      return Promise.reject('Invalid request');
+    });
+    sinon.stub(request, 'patch').callsFake((opts) => {
+      if (opts.url === `https://graph.microsoft.com/v1.0/groupSettings/c391b57d-5783-4c53-9236-cefb5c6ef323` &&
+        JSON.stringify(opts.body) === JSON.stringify({
+          displayName: null,
+          templateId: '62375ab9-6b52-47ed-826b-58e47e0e304b',
+          values: [
+            {
+              name: 'UsageGuidelinesUrl',
+              value: 'https://contoso.sharepoint.com/sites/compliance'
+            },
+            {
+              name: 'ClassificationList',
+              value: 'HBI, MBI, LBI, GDPR'
+            },
+            {
+              name: 'DefaultClassification',
+              value: 'MBI'
+            },
+            {
+              name: 'CustomBlockedWordsList',
+              value: ''
+            },
+            {
+              name: 'EnableMSStandardBlockedWords',
+              value: 'false'
+            },
+            {
+              name: 'ClassificationDescriptions',
+              value: ''
+            },
+            {
+              name: 'PrefixSuffixNamingRequirement',
+              value: ''
+            },
+            {
+              name: 'AllowGuestsToBeGroupOwner',
+              value: 'false'
+            },
+            {
+              name: 'AllowGuestsToAccessGroups',
+              value: 'true'
+            },
+            {
+              name: 'GuestUsageGuidelinesUrl',
+              value: ''
+            },
+            {
+              name: 'GroupCreationAllowedGroupId',
+              value: ''
+            },
+            {
+              name: 'AllowToAddGuests',
+              value: 'true'
+            },
+            {
+              name: 'EnableGroupCreation',
+              value: 'true'
+            }
+          ]
+        })) {
+        return Promise.resolve({
```

Comment:
This response is incorrect. The real patch request to MSGraph doesn't return any value.